src/test/ReactTestUtils.js

```diff
@@ -470,6 +470,11 @@ ReactShallowRenderer.prototype._render = function(element, transaction, context)
 function makeSimulator(eventType) {
   return function(domComponentOrNode, eventData) {
     var node;
+    invariant(
+      !React.isValidElement(domComponentOrNode),
+      'TestUtils.Simulate expects a component instance and not a React Element.' +
+      'TestUtils.Simulate will not work if you are utilizing shallowRender.'
```

Comment:
nit: Instead of "shallowRender" (which isn't actually spelled like that anywhere) can we just say "if you are using shallow rendering"?